src/app.ts

```diff
@@ -13,14 +13,40 @@ interface TemperatureSummary {
   average: number
 }
 
+let reading: TemperatureReading[];
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  reading = readings;
 }
 
 export function getTemperatureSummary(
   date: Date,
   city: string,
 ): TemperatureSummary | null {
-  //add here your code
-  return null
-}
+  let totalTemp: number = 0;
+  let counter: number = 0;
+  let founded: boolean = false;
```

Comment:
In cases where you're assinging values, TS would be smart enough to type the variables correctly.

This would be OK:
```ts
let totalTemp = 0;
let counter = 0;
let founded = false;
```